typedoc.json

```diff
@@ -1,10 +1,9 @@
 {
-    "out": "./docs/",
-    "entryPoints": ["src/index.ts"],
-    "readme": "./README.md",
-    "name": "@top-gg/sdk",
-    "excludeExternals": true,
-    "excludePrivate": true,
-    "excludeInternal": true,
-    "theme": "minimal"
```

Comment:
Can we find a better theme? The default one doesn't work well with such a small library + for some reason the default theme just ignores the Webhook class?

Reply:
> The default one doesn't work well with such a small library

Can you elaborate on what you mean by that? It works really well IMO.

> for some reason the default theme just ignores the Webhook class?

It doesn't ignore anything for me, you might be serving the files incorrectly, try running
```shell
npm run docs && npx http-server docs
```
and go to `http://localhost:8080`, it should work just fine with that.

Reply:
Whoops yeah, I can see the webhook class here now.
I mean we can do it, but I really like how the minimal theme looked, module selectors, and larger settings options seem really unnecessary for a project with two classes and a couple of types.

Reply:
![image](https://user-images.githubusercontent.com/7953076/226073627-b12897a1-e797-4958-8298-7afdd4313729.png)

![image](https://user-images.githubusercontent.com/7953076/226073644-31d3dbc5-631b-4f9b-b7eb-776e4b29b191.png)